### PR TITLE
Feature/get by id performance

### PR DESCRIPTION
**Pull request by Azmat7860**
Sir Please review the code of get Performance by ID API.

### CODE REVIEW

**Comment by Azmat7860** [comment]
Sir please review the code now i think it's fine.

File: models/performance.js [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+        maxlength: 24,
+    },
+    updated_by_id: {
+        type: mongoose.Schema.Types.ObjectId,
+        ref: "Employee",
+        required: true,
+        maxlength: 24,
+    },
+  }
+  // { timestamps: true },
```
Comment (AbubakarZorrain) [inline models/performance.js:42]: Remove these comments...

File: routes/index.js [hunk 2]
```diff
@@ -4,9 +4,11 @@ import express from "express";
 import userRoute from "./user/index.js";
 import attendenceRoute from "./attendence/index.js";
 import miscRoute from "./misc/index.js";
+import performanceRoute from "./performance/index.js";
 import eventRoute from "./event/index.js"
```
Comment (AbubakarZorrain) [inline routes/index.js:9]: Remove empty lines from import.

**Review by AbubakarZorrain — CHANGES_REQUESTED** [review]

**Comment by AbubakarZorrain** [comment]
Resolve conflicts and comments.

**Review by AbubakarZorrain — APPROVED** [review]
Approved...

**Pull request merged**